val.py

```diff
@@ -154,6 +154,9 @@ def run(
 
         # Data
         data = check_dataset(data)  # check
+        
+    channels = data['channels'] if 'channels' in data else []
```

Comment:
What is difference between channels and ch arguments?
It seems ch is for number of channels; but, what about channels? and where is it used?